### PR TITLE
E2E Test: Windows Defender

### PR DESCRIPTION
**Pull request by BelenValdivia**
|Related issue|
|-------------|
|      https://github.com/wazuh/wazuh-qa/issues/3127     |

## Description

The test research was accomplished in https://github.com/wazuh/wazuh-qa/issues/2878. So, in this issue, we develop the E2E test to cover those use cases.

### Added

Windows Defender disable event 

### Configuration options

To launch these tests, we must indicate the path of the inventory:

```
python -m pytest -vvvs tests/end_to_end/test_windows_defender/ --inventory_path=/home/belen/Repositories/wazuh-qa/tests/end_to_end/general_playbooks/inventory.yml 
```
It should be noted that, for local environments, it will be necessary to create the inventory for the moment. It should contain the following:

```
all:
  hosts:
    wazuh-manager:
      ansible_connection: ssh
      ansible_user: vagrant
      ansible_ssh_private_key_file: <path_to_private_key>
      ansible_python_interpreter: /usr/bin/python3
     
    wazuh-windows:
      ansible_user: vagrant
      ansible_password: vagrant
      ansible_connection: winrm
      ansible_winrm_server_cert_validation: ignore
      ansible_winrm_transport: basic
      ansible_winrm_port: 5985
      ansible_python_interpreter: C:\Users\vagrant\AppData\Local\Programs\Pyhton\Python39\python.exe
```

## Testing performed

| Tester             | Test path | Jenkins | Local  | OS | Commit | Notes                |
|--------------------|-----------|---------|--------|-----|--------|----------------------|
| @user (Developer)  |    test_windows_defender/       | :no_entry_sign: :no_entry_sign: :no_entry_sign:|  [🟢](https://github.com/wazuh/wazuh-qa/files/9222205/R1-test-windows-defender.zip) [🟢](https://github.com/wazuh/wazuh-qa/files/9222206/R2-test-windows-defender.zip)  [🟢](https://github.com/wazuh/wazuh-qa/files/9222207/R3-test-windows-defender.zip) |     Windows    |    a2a32cf09b49f318393d5d1265e16a29a635d7b7     | Nothing to highlight |
| @user (Reviewer)   |    test_windows_defender/       | :no_entry_sign: :no_entry_sign: :no_entry_sign:| [🟢 ](https://github.com/wazuh/wazuh-qa/files/9258705/R1-3138-test-windows-defender.zip)[🟢 ](https://github.com/wazuh/wazuh-qa/files/9258707/R2-3138-test-windows-defender.zip)[🟢 ](https://github.com/wazuh/wazuh-qa/files/9258712/R3-3138-test-windows-defender.zip) |    CentOS and Windows    |   dfea381      | Nothing to highlight |

### CODE REVIEW

File: tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        path: C:\temp
+        state: directory
+
+    - name: Copy ossec.conf
+      ansible.windows.win_copy:
+        src: C:\Program Files (x86)\ossec-agent\ossec.conf
+        dest: C:\temp
+        remote_src: true
+
+    - name: Enable the agent module to collect installed packages (Windows)
```
Comment (juliamagan) [inline tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml:16]: Name is not correct

**Review by juliamagan — CHANGES_REQUESTED** [review]

File: tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml (hunk 1 above)
Comment (BelenValdivia) [reply tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml]: Done in f1c9938c96b60a5587ab444a4da2ecd55451d839

File: tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        path: C:\temp
+        state: directory
+
+    - name: Copy ossec.conf
+      ansible.windows.win_copy:
+        src: C:\Program Files (x86)\ossec-agent\ossec.conf
+        dest: C:\temp
+        remote_src: true
+
+    - name: Enable the agent to collect eventchannel logs
```
Comment (juliamagan) [inline tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml:16]: Specify Windows Defender

**Review by juliamagan — CHANGES_REQUESTED** [review]

File: tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml (hunk 2 above)
Comment (BelenValdivia) [reply tests/end_to_end/test_windows_defender/data/playbooks/configuration.yaml]: Done in dfea381fb5e5080842a3344bb57f7b3860228f21

**Review by juliamagan — APPROVED** [review]

**Review by jmv74211 — APPROVED** [review]

**Pull request merged**